pages/DSNP/DSNP-Archive-Messages.mdx

```diff
@@ -0,0 +1,103 @@
+import { Image } from "@chakra-ui/react";
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Proposed |
+
+## Purpose
+1. Specify the Archivist message format
+
+## Assumptions
+* Chain messages are on Ethereum
+* Message data is posted via [Ethereum log events](https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378)
+* Signature algorithm is [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). This allows the use `ecreover`
+to get public keys. A public key also need not be included in a log event for ease of validation.
+* content hashes are created via the same [keccak-256 hashing algorithm](https://en.wikipedia.org/wiki/SHA-3) used by Ethereum.
+
+## Archive Entry
+An archive entry is a combination of data in a DSNP message and from the block in which it is included.
+It is a key-value map consisting of the following fields:
+
+|field|description|type|required?|
+|---|---|---|---|
+| `author` | social identity of author of the content. | bytes | YES
+| `blockHeight` | the block number this message was included in | number | YES
+| `fromAddress`| social identity of message sender | bytes | YES
+| `logIndex` | the index within the logs of this message | number | YES
+| `messageID` | keccak-256 hash of this data, minus the messageID |  bytes3 | YES
+| `dsnpType` | DSNP message type |number/enum. see [DSNP Message Types](/DSNP/DSNP-Message-Types) | YES
+| `dsnpData` | DSNP message data | see fields in [DSNP Messages](/DSNP/DSNP-Messages) | YES
+| `signatures` | list of additional signatures referencing this message | array of Signatures | YES
+| `transactionIndex` | the index of the transaction this message is associated with | number | YES
+
+## FIELDS DETAILS
+
+### `author`
+* bytes
+* The social identity of the composer of the message data. This may be the same as `fromAddress`, however, if the author is different, it means the message sender is a delegated sender of this message, authorized by `author`.  Validators may opt to verify that the sender is authorized, however, this will be enforced by the social identity contract.
+
+### `blockHeight`
+* number
+* The block in which this DSNP Message is included.
+
+### `dsnpType`
+   * number
+   * Indicates what type of message this is, useful for indexers and filters.
+
+### `dsnpData`
+   * varies
+   * This can be encrypted, and/or serialized, and compressed where feasible.  The decrypted, fully deserialzed version must be one of the types described in [DSNP Messages](/DSNP/DSNP-Messages).
+
+### `fromAddress`
+* bytes
+* The contract address of the message sender.  This may be the same as `author`. If different from    `author`, it means the message sender is a delegated sender of this message, authorized by `author`.  Validators may opt to verify that the sender is authorized, however, this will be enforced by the social identity contract.
```

Comment:
@shannonwells Let's talk about this. I think we might need to get a clearer distinction between the signature on behalf of the author and the author.